crates/cargo-winrt/src/main.rs

```diff
@@ -169,32 +170,53 @@ fn verbose() -> bool {
     VERBOSITY.get().copied().unwrap_or(false)
 }
 
-macro_rules! debug {
-    ($($arg:tt)*) => {
-        if verbose() {
-            eprintln!("\t{}", format!($($arg)*));
-        }
-    };
+/// Formats the elapsed time to match Cargo's output.
+fn elapsed(duration: Duration) -> String {
+    let secs = duration.as_secs();
+
+    if secs >= 60 {
+        format!("{}m {:02}s", secs / 60, secs % 60)
+    } else {
+        format!("{}.{:02}s", secs, duration.subsec_nanos() / 10_000_000)
+    }
+}
+
+fn print_status<D>(status: &str, message: D)
```

Comment:
I think I would prefer these as macros so you're not required to do the funny `format_args` or `format` dance. Is there a reason to not use macros for this?

Reply:
I was mostly borrowing from the way Cargo does this output, while maintaining the semantics already used here, so it wasn't explicitly a decision to 'not use macros'. Happy to give it a shot with macros if you think that'd be more useful.